pkg/bliss/httpfx/uris/common-path.go

```diff
@@ -26,6 +29,7 @@ func CommonPath(p1, p2 *Pattern) string {
 			writeSegment(&b, s1)
 		}
 	}
+	fmt.Println(segs1, segs2)
```

Comment:
```suggestion
```

I think it's forgotten during development/testing. Not a big deal though, we can always erase it after the merge.